src/getUserDataPath.ts

```diff
@@ -0,0 +1,12 @@
+import * as vscode from "vscode";
+import * as path from "path";
+
+export function getUserDataPath(context: vscode.ExtensionContext): string {
+  const userPath = path.resolve(context.globalStorageUri.fsPath, "../..");
```

Comment:
is there not a more reliable way to do this? seems slightly brittle, though I'd imagine they're very unlikely to change this at any time in the future? Assuming it works on all platforms?

Reply:
I didn't find any unfortunately. That's why the if statement is there as a sanity check.

This should work on all platforms but I can't really verify that for myself.